tests/unit/steps/test_steps_plan.py

```diff
@@ -216,15 +222,107 @@ def test_determine_upgrade_target_no_upgrade_available():
         ),  # current_series is None
     ],
 )
-def test_determine_upgrade_target_invalid_input(current_os_release, current_series, exp_error_msg):
+def test_pre_plan_sane_checks_is_valid_openstack_cloud(
+    current_os_release, current_series, exp_error_msg, cli_args
+):
     with pytest.raises(NoTargetError, match=exp_error_msg):
-        determine_upgrade_target(current_os_release, current_series)
+        mock_analysis_result = MagicMock(spec=Analysis)()
+        mock_analysis_result.current_cloud_series = current_series
+        mock_analysis_result.current_cloud_os_release = current_os_release
+        pre_plan_sane_checks(cli_args, mock_analysis_result)
+
+
+@pytest.mark.parametrize(
+    "current_os_release, current_series",
+    [
+        (OpenStackRelease("yoga"), "jammy"),
+        (OpenStackRelease("train"), "bionic"),
+    ],
+)
+def test_pre_plan_sane_checks_is_supported_series(current_os_release, current_series, cli_args):
+    mock_analysis_result = MagicMock(spec=Analysis)()
+    with pytest.raises(OutOfSupportRange):
```

Comment:
We should try to match also error message in any pytest.raises (here and in other tests), which is checking our own error with custom message.